apps/yara/CMakeLists.txt

```diff
@@ -14,22 +14,14 @@ message (STATUS "Configuring apps/yara")
 # Dependencies
 # ----------------------------------------------------------------------------
 
-# Disable older MSVC.
-if (MSVC AND MSVC_VERSION LESS 1800)
-    message (STATUS "  Visual Studio 2012 or newer required to build yara.")
-    return ()
-endif ()
-
-# Disable parallelism in MSVC as it supports only OpenMP 2.0.
-if (NOT MSVC)
```

Comment:
I had thought that vs2015 had openmp3, but apparently its not the case :facepunch: 

Well, since we have global detection now (for caching), it cannot be disabled locally, without doing major regex voodoo to all the build vars.
Can I just deactivate builds entirely for msvc? Nobody would seriously use it without threading anyway, would they?